docs/spec_draft.md

```diff
@@ -941,6 +942,69 @@ produces an implementation-defined value.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.dynamic_update_slice
+
+### Semantics
+
+Overwrites the `operand` tensor at `start_indices` with values from `update`
+tensor and produces a `result` tensor.
+
+More formally, `result[j0, ..., jR-1]` is defined as:
+  * `update[i0, ..., iR-1]` if `jd = effective_start_indices[d] + id` where
+    `effective_start_indices[d] = clamp(start_indices[d], 0, dim(operand, d) - `
+    `dim(update, d))`
```

Comment:
Let's reword similarly to https://github.com/openxla/stablehlo/pull/556#discussion_r1030778738.

Reply:
Lmk if the updated wording is okay.